mod_reforged/hooks/entity/tactical/enemies/skeleton_heavy_bodyguard.nut

```diff
@@ -1,7 +1,7 @@
 ::Reforged.HooksMod.hook("scripts/entity/tactical/enemies/skeleton_heavy_bodyguard", function(q) {
 	q.onInit = @() function()
 	{
-	   this.skeleton.onInit();
+	    this.skeleton.onInit();
```

Comment:
if you feel like fixing indentation then go ahead and outfit the line directly with tabulator